.travis.yml

```diff
@@ -52,8 +52,9 @@ script:
   - |
       if [[ $TESTS == 'blocks-examples' ]]; then
         cd $TRAVIS_BUILD_DIR
-        git clone https://github.com/mila-udem/blocks-examples.git
+        git clone https://github.com/laurent-dinh/blocks-examples.git
```

Comment:
The associated PR in blocks-example can be found [here](https://github.com/mila-udem/blocks-examples/pull/16).